docs/docs/workspaces/sso/README.md

```diff
@@ -0,0 +1,67 @@
+# Single Sign-On Overview
+
+Pipedream supports Single Sign-On (SSO) using SAML or Google OAuth, which allows IT and team administrators easier controls to manage access and security.
```

Comment:
maybe "using SAML or Google OAuth" -> "with Okta, Google, or any provider that supports SAML"? And then link to each of the sub-docs.

Reply:
"team administrators" -> "workspace administrators"?